src/core/packages/plugins/server-internal/src/plugins_config.ts

```diff
@@ -21,6 +26,32 @@ const configSchema = schema.object({
    * Defines an array of directories where another plugin should be loaded from.
    */
   paths: schema.arrayOf(schema.string(), { defaultValue: [] }),
+  /**
+   * Defines an array of groups to ignore when loading plugins.
+   */
+  excludedPluginGroups: schema.maybe(
+    schema.arrayOf(
+      schema.oneOf(
+        KIBANA_SOLUTIONS.map((solutionName) => schema.literal(solutionName)) as [
+          Type<KibanaSolution> // This cast is needed because it's different to Type<T>[] :sight:
+        ]
+      ),
+      { defaultValue: [] }
+    )
+  ),
+  /**
+   * Defines an array of groups to NOT ignore when loading plugins.
+   */
+  includedPluginGroups: schema.maybe(
+    schema.arrayOf(
+      schema.oneOf(
+        KIBANA_SOLUTIONS.map((solutionName) => schema.literal(solutionName)) as [
+          Type<KibanaSolution> // This cast is needed because it's different to Type<T>[] :sight:
+        ]
+      ),
+      { defaultValue: [] }
```

Comment:
If we omit the default, can we assume that `includedPluginGroups = undefined` means no filtering applies? So `undefined` means include all.